File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task69_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        printf("Usage: %s <file_path>\\n", argv[0]);
        return 1;
    }

    const char *filePath = argv[1];

    if (remove(filePath) == 0) {
        printf("File '%s' has been deleted successfully.\\n", filePath);
    } else {
        perror("Error");
        printf("Unable to delete '%s'.\\n", filePath);
    }

    return 0;
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task69_CLAUDE_claude-3-5-sonnet-20240620.java

import java.io.File;

public class DeleteFile {
    public static void main(String[] args) {
        if (args.length != 1) {
            System.out.println("Usage: java DeleteFile <file_path>");
            return;
        }

        String filePath = args[0];
        File file = new File(filePath);

        if (file.delete()) {
            System.out.println("File deleted successfully: " + filePath);
        } else {
            System.out.println("Failed to delete the file: " + filePath);
        }
    }
}
